store/sqlstore/thread_store.go

```diff
@@ -359,12 +359,25 @@ func (s *SqlThreadStore) GetThreadsForUser(userId, teamId string, opts model.Get
 	return result, nil
 }
 
-func (s *SqlThreadStore) GetThreadFollowers(threadID string) ([]string, error) {
+func (s *SqlThreadStore) GetThreadFollowers(threadID string, fetchOnlyActive bool) ([]string, error) {
 	var users []string
+
+	fetchConditions := sq.And{
```

Comment:
I don't think you need the `sq.And` here since it's just the one condition

Reply:
@jwilander I get `Incompatible assign` in the re-assignment below if I don't add the `sq.And` when declaring.